examples/consumer/main.go

```diff
@@ -249,18 +291,22 @@ func createSubscription(resourceAddress string) (sub pubsub.PubSub, status int,
 }
 
 // getCurrentState get event state for the resource
-func getCurrentState(resource string) {
+func getCurrentState(resource string) error {
 	//create publisher
 	url := &types.URI{URL: url.URL{Scheme: "http",
 		Host: apiAddr,
 		Path: fmt.Sprintf("%s%s", apiPath, fmt.Sprintf("%s/CurrentState", resource[1:]))}}
 	rc := restclient.New()
-	status, cloudEvent := rc.Get(url)
+	status, cloudEvent, err := rc.Get(url)
 	if status != http.StatusOK {
-		log.Errorf("CurrentState:error %d from url %s, %s", status, url.String(), cloudEvent)
+		if err != nil {
+			log.Errorf("%s", err.Error())
```

Comment:
Couldn't this be log.Error(err) or if it needs to be a string log.Error(err.Error())